scripts/lib/CIME/XML/env_batch.py

```diff
@@ -65,7 +65,7 @@ def set_value(self, item, value, subgroup=None, ignore_type=False):
         return val
 
     # pylint: disable=arguments-differ
-    def get_value(self, item, attribute=None, resolved=True, subgroup="case.run"):
+    def get_value(self, item, attribute=None, resolved=True, subgroup="PRIMARY"):
```

Comment:
This is a little hacky. No case object is available to retrieve the primary job id.